src/components/GetUser.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import Typography from '@material-ui/core/Typography';
+import User from './User';
+import RegisteredUser from './RegisteredUser';
+import GetUserForm from './GetUserForm';
+import RegisterUserForm from './RegisterUserForm';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = {
+  text: {
+    marginTop: 20,
+  },
+};
+
+class GetUser extends Component {
+  constructor (props) {
+    super(props);
+
+    this.state = {
```

Comment:
You don't need these state properties. Please remove

Reply:
Hi Max, copy/paste leftovers -- removed.